src/components/Form/FormField.tsx

```diff
@@ -0,0 +1,188 @@
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import { UIComponent, customPropTypes, childrenExist, createShorthandFactory } from '../../lib'
+import { ComponentVariablesInput, ComponentPartStyle } from '../../../types/theme'
+import {
+  Extendable,
+  ReactChildren,
+  ShorthandValue,
+  ShorthandRenderFunction,
+} from '../../../types/utils'
+import Text from '../Text'
+import { default as Slot } from '../Slot/Slot'
+import Input from '../Input'
+
+export interface IFormFieldProps {
+  as?: any
+  children?: ReactChildren
```

Comment:
this is not correct type - here is the explanation of that: https://github.com/stardust-ui/react/pull/328#discussion_r223225907

Reply:
Same as above